cobigen/cobigen-core-systemtest/src/test/java/com/devonfw/cobigen/systemtest/TemplateProcessingTest.java

```diff
@@ -62,21 +85,23 @@ public void extractTemplateSetsTest() throws IOException {
         .resolve(ConfigurationConstants.ADAPTED_FOLDER);
     Path extractedJar1 = adaptedFolder.resolve("template-test1-0.0.1");
     Path extractedJar2 = adaptedFolder.resolve("template-test2-0.0.1");
-    assertThat(Files.exists(extractedJar1));
-    assertThat(Files.exists(extractedJar2));
+    assertThat(extractedJar1.toFile()).exists().isDirectory();
+    assertThat(extractedJar2.toFile()).exists().isDirectory();
   }
 
   /**
    * @throws IOException if an Exception occurs
    */
+  @Test
   public void extractTemplatesWithOldConfiguration() throws IOException {
 
-    Path cobigenTemplatesProject = this.cobiGenHome.toPath()
-        .resolve(ConfigurationConstants.CONFIG_PROPERTY_TEMPLATES_PATH)
-        .resolve(ConfigurationConstants.COBIGEN_TEMPLATES);
+    Path cobigenTemplatesParent = this.cobiGenHome.toPath()
```

Comment:
Can't you use `CobiGenPaths.getCobiGenHomePath()` instead of `this.cobiGenHome.toPath()`?